docs/index.html

```diff
@@ -2226,8 +2230,8 @@ <h2 id="_run_openqa_workers">Run openQA workers</h2>
 </div>
 <div class="paragraph">
 <p>This will start worker number one. You can start as
-many workers as you dare, you just need to supply different 'worker id' (number
-after @).</p>
+many workers as you dare, you just need to supply different 'instance number'
```

Comment:
```suggestion
many workers as you dare, you just need to supply a different 'instance number'
```

I guess there's little pointing in questing changes here, but in case somebody sees this and touches the code again...

Reply:
what you can do is create a github issue from this comment in the three-dot menue top right or right away click the pencil icon to draft a change in the webUI editor.